docs/source/reference/modding/settings/game.rst

```diff
@@ -282,3 +282,19 @@ equivalent to the one introduced by the equivalent Morrowind Code Patch feature.
 This makes the movement speed behavior more fair between different races.
 
 This setting can be controlled in Advanced tab of the launcher.
+
+projectiles enchant multiplier
+------------------------------
+
+:Type:		floating point
+:Range:		0.0 to 1.0
+:Default:	0.0
+
+This setting determines which percentage of soulgem charge can be used to enchant multiple projectiles (thrown weapons, arrows and bolts):
```

Comment:
The value of this setting determines how many projectiles (thrown weapons, arrows and bolts) you can enchant at once according to the following formula: